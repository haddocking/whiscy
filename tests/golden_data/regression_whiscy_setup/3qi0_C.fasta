>3qi0_C.fasta
TSVVAWGGNNDWGEATVPAEAQSGVDAIAGGYFHGLALKGGKVLGWGANLNGQLTMPAAT
QSGVDAIAAGNYHSLALKDGEVIAWGGNEDGQTTVPAEARSGVDAIAAGAWASYALKDGK
VIAWGDDSDGQTTVPAEAQSGVTALDGGVYTALAVKNGGVIAWGDNYFGQTTVPAEAQSG
VDDVAGGIFHSLALKDGKVIAWGDNRYKQTTVPTEALSGVSAIASGEWYSLALKNGKVIA
WGSSRTAPSSVQSGVSSIEAGPNAAYALK
