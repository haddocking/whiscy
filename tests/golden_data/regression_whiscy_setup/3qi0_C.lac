55
70
71
72
79
84
85
109
111
118
121
123
124
126
128
133
148
150
151
153
155
157
162
163
165
172
188
189
190
192
193
194
196
200
204
206
211
225
226
227
228
229
231
232
235
240
241
243
245
250
265
267
270
271
274
279
280
282
283
302
305
307
309
