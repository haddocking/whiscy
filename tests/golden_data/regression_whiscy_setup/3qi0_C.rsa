REM  FreeSASA 2.0.3
REM  Absolute and relative SASAs for 3qi0_C.pdb
REM  Atomic radii and reference values for relative SASA: NACCESS
REM  Chains: C
REM  Algorithm: Lee & Richards
REM  Probe-radius: 1.40
REM  Slices: 20
REM RES _ NUM      All-atoms   Total-Side   Main-Chain    Non-polar    All polar
REM                ABS   REL    ABS   REL    ABS   REL    ABS   REL    ABS   REL
RES THR C  42   131.21  94.2  86.52  88.7  44.69 107.2  56.04  74.0  75.17 118.4
RES SER C  43    40.23  34.5  38.97  53.3   1.26   2.9  25.21  51.9  15.02  22.1
RES VAL C  44    17.80  11.8   0.22   0.2  17.59  42.7   0.22   0.2  17.59  48.7
RES VAL C  45    30.85  20.4  28.80  26.1   2.05   5.0  30.85  26.8   0.00   0.0
RES ALA C  46    21.54  20.0   0.47   0.7  21.07  47.9   0.47   0.7  21.07  57.4
RES TRP C  47    33.01  13.3  24.76  12.0   8.25  19.5  33.01  17.5   0.00   0.0
RES GLY C  48    19.93  24.8   0.00   N/A  19.93  24.8   7.94  21.1  11.99  28.1
RES GLY C  49    46.81  58.3   0.00   N/A  46.81  58.3  18.80  49.9  28.01  65.7
RES ASN C  50   123.93  86.1  89.42  86.9  34.52  84.1  22.40  48.5 101.53 103.8
RES ASN C  51    53.60  37.2  39.18  38.1  14.41  35.1  14.56  31.5  39.04  39.9
RES ASP C  52    89.71  63.9  53.80  54.5  35.91  86.0  22.71  46.1  66.99  73.5
RES TRP C  53    88.32  35.5  75.87  36.7  12.45  29.4  62.83  33.2  25.49  42.6
RES GLY C  54    22.57  28.1   0.00   N/A  22.57  28.1  18.24  48.4   4.33  10.2
RES GLU C  55     4.89   2.8   4.89   3.8   0.00   0.0   1.99   3.3   2.90   2.6
RES ALA C  56    18.67  17.3   6.30   9.8  12.37  28.2   6.30   8.8  12.37  33.7
RES THR C  57    81.79  58.7  74.43  76.3   7.36  17.7  67.82  89.5  13.98  22.0
RES VAL C  58    22.41  14.8  11.00  10.0  11.41  27.7  11.00   9.5  11.41  31.6
RES PRO C  59    30.55  22.5  19.24  17.7  11.31  41.7  30.55  25.3   0.00   0.0
RES ALA C  60    84.36  78.2  63.88  99.9  20.47  46.6  65.40  91.9  18.96  51.6
RES GLU C  61    95.34  55.4  83.94  64.4  11.40  27.3  52.88  88.2  42.46  37.9
RES ALA C  62     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLN C  63    88.05  49.4  83.19  60.9   4.86  11.6  26.93  51.9  61.12  48.4
RES SER C  64    57.78  49.6  55.61  76.0   2.17   5.0  26.10  53.8  31.68  46.6
RES GLY C  65    40.49  50.4   0.00   N/A  40.49  50.4  31.98  84.8   8.51  20.0
RES VAL C  66    17.10  11.3   0.60   0.5  16.50  40.1   0.72   0.6  16.38  45.3
RES ASP C  67    88.32  62.9  62.43  63.2  25.89  62.0  28.39  57.6  59.93  65.7
RES ALA C  68    31.49  29.2  29.40  46.0   2.09   4.8  31.49  44.2   0.00   0.0
RES ILE C  69     8.60   4.9   0.08   0.1   8.53  20.7   0.08   0.1   8.53  23.6
RES ALA C  70     2.62   2.4   2.62   4.1   0.00   0.0   2.62   3.7   0.00   0.0
RES GLY C  71     5.90   7.4   0.00   N/A   5.90   7.4   0.00   0.0   5.90  13.9
RES GLY C  72     2.04   2.5   0.00   N/A   2.04   2.5   0.09   0.2   1.95   4.6
RES TYR C  73    56.77  26.8  55.74  32.0   1.03   2.7  21.31  15.7  35.47  46.5
RES PHE C  74    29.08  14.6  29.08  18.0   0.00   0.0  29.08  17.6   0.00   0.0
RES HIS C  75     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLY C  76     0.00   0.0   0.00   N/A   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C  77     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES ALA C  78     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C  79     7.87   4.4   7.87   5.7   0.00   0.0   7.87   5.5   0.00   0.0
RES LYS C  80    72.64  36.3  67.07  42.3   5.57  13.4  42.94  37.0  29.70  35.2
RES GLY C  81    71.80  89.4   0.00   N/A  71.80  89.4  44.18 117.2  27.62  64.8
RES GLY C  82    13.19  16.4   0.00   N/A  13.19  16.4   8.24  21.9   4.96  11.6
RES LYS C  83    85.36  42.6  85.36  53.9   0.00   0.0  36.85  31.8  48.50  57.5
RES VAL C  84     2.01   1.3   0.00   0.0   2.01   4.9   0.00   0.0   2.01   5.6
RES LEU C  85    15.49   8.7  11.65   8.4   3.85   9.7  15.49  10.9   0.00   0.0
RES GLY C  86    21.08  26.2   0.00   N/A  21.08  26.2   2.26   6.0  18.81  44.1
RES TRP C  87    17.71   7.1  11.02   5.3   6.69  15.8  17.71   9.4   0.00   0.0
RES GLY C  88    34.17  42.5   0.00   N/A  34.17  42.5   4.31  11.4  29.85  70.0
RES ALA C  89    19.02  17.6  12.88  20.1   6.14  14.0  19.02  26.7   0.00   0.0
RES ASN C  90    74.61  51.8  46.61  45.3  28.00  68.2  22.63  49.1  51.98  53.1
RES LEU C  91   112.30  62.9  78.25  56.3  34.05  86.2  82.59  58.2  29.72  81.5
RES ASN C  92    43.92  30.5  21.32  20.7  22.60  55.1  15.94  34.5  27.98  28.6
RES GLY C  93    25.35  31.6   0.00   N/A  25.35  31.6  20.92  55.5   4.43  10.4
RES GLN C  94     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C  95    30.15  16.9  14.33  10.3  15.82  40.1  14.33  10.1  15.82  43.4
RES THR C  96    88.94  63.9  79.08  81.1   9.86  23.6  71.73  94.7  17.20  27.1
RES MET C  97    45.52  23.5  18.75  12.3  26.77  64.2  19.12  12.2  26.40  72.4
RES PRO C  98    38.63  28.4  26.99  24.8  11.63  42.9  38.63  32.0   0.00   0.0
RES ALA C  99    91.91  85.2  68.11 106.5  23.80  54.2  73.16 102.8  18.75  51.1
RES ALA C 100    48.87  45.3  36.65  57.3  12.22  27.8  44.82  63.0   4.05  11.0
RES THR C 101     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLN C 102    76.18  42.7  65.68  48.1  10.50  25.2  34.49  66.5  41.68  33.0
RES SER C 103    67.28  57.7  66.39  90.7   0.88   2.0  37.47  77.2  29.81  43.8
RES GLY C 104    40.23  50.1   0.00   N/A  40.23  50.1  35.35  93.8   4.88  11.5
RES VAL C 105     7.15   4.7   0.00   0.0   7.15  17.4   0.00   0.0   7.15  19.8
RES ASP C 106    85.76  61.0  58.46  59.2  27.30  65.4  27.14  55.1  58.63  64.3
RES ALA C 107    26.35  24.4  24.26  37.9   2.09   4.8  26.35  37.0   0.00   0.0
RES ILE C 108    20.14  11.5   0.93   0.7  19.21  46.7   0.93   0.7  19.21  53.2
RES ALA C 109    10.46   9.7   7.34  11.5   3.11   7.1  10.46  14.7   0.00   0.0
RES ALA C 110    14.49  13.4   0.00   0.0  14.49  33.0   0.00   0.0  14.49  39.5
RES GLY C 111     3.20   4.0   0.00   N/A   3.20   4.0   3.20   8.5   0.00   0.0
RES ASN C 112    16.39  11.4  16.39  15.9   0.00   0.0   0.55   1.2  15.84  16.2
RES TYR C 113    67.33  31.7  67.33  38.7   0.00   0.0  27.20  20.0  40.13  52.6
RES HIS C 114     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES SER C 115     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 116     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES ALA C 117     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 118     7.98   4.5   7.98   5.7   0.00   0.0   7.98   5.6   0.00   0.0
RES LYS C 119    52.92  26.4  49.77  31.4   3.15   7.6  29.60  25.5  23.32  27.7
RES ASP C 120   119.02  84.7  84.87  86.0  34.15  81.8  45.49  92.3  73.53  80.6
RES GLY C 121    10.46  13.0   0.00   N/A  10.46  13.0   5.57  14.8   4.89  11.5
RES GLU C 122    71.31  41.4  71.15  54.6   0.16   0.4  15.50  25.8  55.82  49.8
RES VAL C 123     2.80   1.8   0.00   0.0   2.80   6.8   0.00   0.0   2.80   7.7
RES ILE C 124    22.67  12.9  19.86  14.8   2.81   6.8  22.67  16.3   0.00   0.0
RES ALA C 125    17.35  16.1   0.00   0.0  17.35  39.5   0.00   0.0  17.35  47.3
RES TRP C 126    16.92   6.8  14.07   6.8   2.86   6.7  16.92   8.9   0.00   0.0
RES GLY C 127    20.95  26.1   0.00   N/A  20.95  26.1   0.58   1.5  20.37  47.8
RES GLY C 128     4.50   5.6   0.00   N/A   4.50   5.6   4.50  11.9   0.00   0.0
RES ASN C 129    63.87  44.4  44.40  43.1  19.48  47.5  18.79  40.7  45.09  46.1
RES GLU C 130   127.01  73.8  92.08  70.6  34.93  83.7  71.14 118.6  55.87  49.8
RES ASP C 131    61.19  43.6  40.89  41.4  20.29  48.6  16.71  33.9  44.48  48.8
RES GLY C 132    25.49  31.7   0.00   N/A  25.49  31.7  21.28  56.5   4.21   9.9
RES GLN C 133     0.65   0.4   0.65   0.5   0.00   0.0   0.00   0.0   0.65   0.5
RES THR C 134    28.60  20.5  23.27  23.8   5.33  12.8  23.27  30.7   5.33   8.4
RES THR C 135    85.28  61.2  76.60  78.5   8.68  20.8  68.00  89.8  17.28  27.2
RES VAL C 136    14.99   9.9   8.68   7.9   6.31  15.3   8.68   7.5   6.31  17.5
RES PRO C 137    42.41  31.2  33.87  31.1   8.54  31.5  42.41  35.1   0.00   0.0
RES ALA C 138    81.89  75.9  63.60  99.5  18.29  41.6  65.03  91.4  16.86  45.9
RES GLU C 139    75.26  43.7  63.62  48.8  11.64  27.9  45.11  75.2  30.15  26.9
RES ALA C 140     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES ARG C 141   128.81  54.0 121.93  62.0   6.88  16.5  38.42  49.7  90.39  56.1
RES SER C 142    65.75  56.4  65.71  89.8   0.04   0.1  36.87  76.0  28.89  42.5
RES GLY C 143    39.26  48.9   0.00   N/A  39.26  48.9  33.03  87.6   6.24  14.6
RES VAL C 144     7.98   5.3   0.00   0.0   7.98  19.4   0.00   0.0   7.98  22.1
RES ASP C 145    84.21  59.9  57.41  58.2  26.79  64.2  32.19  65.3  52.02  57.0
RES ALA C 146    34.98  32.4  32.00  50.0   2.98   6.8  34.98  49.2   0.00   0.0
RES ILE C 147    20.42  11.7   1.22   0.9  19.20  46.6   1.22   0.9  19.20  53.2
RES ALA C 148     7.56   7.0   5.44   8.5   2.12   4.8   7.56  10.6   0.00   0.0
RES ALA C 149    14.70  13.6   1.07   1.7  13.63  31.0   1.07   1.5  13.63  37.1
RES GLY C 150     4.48   5.6   0.00   N/A   4.48   5.6   4.48  11.9   0.00   0.0
RES ALA C 151     5.94   5.5   0.73   1.1   5.21  11.9   3.82   5.4   2.12   5.8
RES TRP C 152    55.37  22.2  55.08  26.7   0.29   0.7  50.34  26.6   5.02   8.4
RES ALA C 153     1.50   1.4   1.50   2.3   0.00   0.0   1.50   2.1   0.00   0.0
RES SER C 154     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES TYR C 155     1.58   0.7   1.58   0.9   0.00   0.0   1.05   0.8   0.54   0.7
RES ALA C 156     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 157    12.16   6.8  12.16   8.8   0.00   0.0  12.16   8.6   0.00   0.0
RES LYS C 158    47.71  23.8  45.13  28.5   2.58   6.2  22.43  19.4  25.28  30.0
RES ASP C 159   113.62  80.9  81.04  82.1  32.58  78.0  46.81  95.0  66.81  73.3
RES GLY C 160    13.39  16.7   0.00   N/A  13.39  16.7   7.77  20.6   5.62  13.2
RES LYS C 161    66.58  33.3  66.58  42.0   0.00   0.0  29.41  25.4  37.17  44.1
RES VAL C 162     1.45   1.0   0.06   0.1   1.39   3.4   0.06   0.1   1.39   3.9
RES ILE C 163    11.59   6.6   7.90   5.9   3.70   9.0  11.59   8.3   0.00   0.0
RES ALA C 164    17.30  16.0   0.00   0.0  17.30  39.4   0.00   0.0  17.30  47.1
RES TRP C 165    17.52   7.0  15.54   7.5   1.99   4.7  17.52   9.3   0.00   0.0
RES GLY C 166    16.74  20.8   0.00   N/A  16.74  20.8   0.00   0.0  16.74  39.3
RES ASP C 167    21.32  15.2  18.55  18.8   2.78   6.6   4.37   8.9  16.96  18.6
RES ASP C 168    61.35  43.7  45.20  45.8  16.15  38.7  23.58  47.8  37.77  41.4
RES SER C 169    93.22  80.0  52.83  72.2  40.39  93.1  53.14 109.5  40.07  58.9
RES ASP C 170    48.40  34.5  32.36  32.8  16.03  38.4  19.07  38.7  29.32  32.2
RES GLY C 171    27.93  34.8   0.00   N/A  27.93  34.8  22.16  58.8   5.77  13.5
RES GLN C 172     1.74   1.0   1.74   1.3   0.00   0.0   1.37   2.6   0.38   0.3
RES THR C 173    27.97  20.1  19.69  20.2   8.28  19.9  19.83  26.2   8.14  12.8
RES THR C 174    84.59  60.7  76.13  78.0   8.46  20.3  68.64  90.6  15.95  25.1
RES VAL C 175    26.10  17.2   8.62   7.8  17.48  42.5   8.62   7.5  17.48  48.4
RES PRO C 176    37.61  27.7  26.54  24.4  11.06  40.8  37.61  31.2   0.00   0.0
RES ALA C 177    83.01  76.9  63.56  99.4  19.45  44.3  66.46  93.4  16.55  45.1
RES GLU C 178   109.09  63.4 106.94  82.0   2.16   5.2  52.33  87.3  56.76  50.6
RES ALA C 179     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLN C 180    80.97  45.4  71.60  52.4   9.37  22.5  14.50  27.9  66.47  52.6
RES SER C 181    60.08  51.5  59.64  81.5   0.44   1.0  37.78  77.8  22.30  32.8
RES GLY C 182    38.58  48.1   0.00   N/A  38.58  48.1  29.34  77.9   9.24  21.7
RES VAL C 183     6.34   4.2   0.00   0.0   6.34  15.4   0.00   0.0   6.34  17.5
RES THR C 184   103.34  74.2  74.70  76.6  28.64  68.7  67.15  88.6  36.19  57.0
RES ALA C 185    16.09  14.9  10.58  16.5   5.51  12.5  16.09  22.6   0.00   0.0
RES LEU C 186    22.34  12.5   5.15   3.7  17.19  43.5   5.15   3.6  17.19  47.2
RES ASP C 187    34.41  24.5  34.01  34.5   0.40   1.0   8.87  18.0  25.54  28.0
RES GLY C 188     8.95  11.1   0.00   N/A   8.95  11.1   0.00   0.0   8.95  21.0
RES GLY C 189     5.45   6.8   0.00   N/A   5.45   6.8   4.33  11.5   1.12   2.6
RES VAL C 190    14.42   9.5  12.36  11.2   2.06   5.0  12.51  10.8   1.91   5.3
RES TYR C 191    62.29  29.4  62.29  35.8   0.00   0.0  28.13  20.7  34.16  44.7
RES THR C 192     0.72   0.5   0.41   0.4   0.32   0.8   0.00   0.0   0.72   1.1
RES ALA C 193     0.88   0.8   0.88   1.4   0.00   0.0   0.88   1.2   0.00   0.0
RES LEU C 194     0.09   0.1   0.00   0.0   0.09   0.2   0.00   0.0   0.09   0.3
RES ALA C 195     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES VAL C 196     1.35   0.9   1.35   1.2   0.00   0.0   1.35   1.2   0.00   0.0
RES LYS C 197    39.17  19.6  33.30  21.0   5.88  14.1  19.93  17.2  19.25  22.8
RES ASN C 198   125.55  87.2  90.19  87.6  35.35  86.2  43.67  94.7  81.87  83.7
RES GLY C 199    19.06  23.7   0.00   N/A  19.06  23.7  11.40  30.2   7.66  18.0
RES GLY C 200    10.73  13.4   0.00   N/A  10.73  13.4  10.73  28.5   0.00   0.0
RES VAL C 201     9.82   6.5   0.00   0.0   9.82  23.9   0.00   0.0   9.82  27.2
RES ILE C 202    24.11  13.8  21.20  15.8   2.91   7.1  24.11  17.3   0.00   0.0
RES ALA C 203    18.14  16.8   0.00   0.0  18.14  41.3   0.00   0.0  18.14  49.4
RES TRP C 204    17.69   7.1  13.62   6.6   4.07   9.6  17.69   9.4   0.00   0.0
RES GLY C 205    20.53  25.6   0.00   N/A  20.53  25.6   0.38   1.0  20.15  47.3
RES ASP C 206    11.91   8.5   8.22   8.3   3.69   8.8   4.50   9.1   7.41   8.1
RES ASN C 207    76.47  53.1  53.86  52.3  22.61  55.1  24.91  54.0  51.56  52.7
RES TYR C 208   185.84  87.6 158.60  91.1  27.23  71.4 118.84  87.5  67.00  87.8
RES PHE C 209    68.42  34.3  50.61  31.4  17.81  46.7  59.41  36.0   9.01  26.3
RES GLY C 210    28.64  35.7   0.00   N/A  28.64  35.7  23.37  62.0   5.27  12.4
RES GLN C 211     0.14   0.1   0.14   0.1   0.00   0.0   0.14   0.3   0.00   0.0
RES THR C 212    16.49  11.8   3.89   4.0  12.60  30.2   1.04   1.4  15.45  24.3
RES THR C 213    75.03  53.9  66.32  68.0   8.72  20.9  57.06  75.3  17.98  28.3
RES VAL C 214    34.83  23.0  18.49  16.8  16.34  39.7  18.49  16.0  16.34  45.2
RES PRO C 215    33.29  24.5  23.77  21.9   9.53  35.2  33.29  27.6   0.00   0.0
RES ALA C 216    84.96  78.7  63.06  98.6  21.90  49.8  65.01  91.4  19.94  54.3
RES GLU C 217    78.39  45.6  65.48  50.2  12.90  30.9  43.87  73.2  34.52  30.8
RES ALA C 218     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLN C 219   104.29  58.5  88.15  64.6  16.14  38.7  36.44  70.2  67.86  53.7
RES SER C 220    63.33  54.3  62.42  85.3   0.91   2.1  34.56  71.2  28.78  42.3
RES GLY C 221    41.63  51.8   0.00   N/A  41.63  51.8  29.99  79.6  11.64  27.3
RES VAL C 222    11.72   7.7   0.00   0.0  11.72  28.5   0.78   0.7  10.94  30.3
RES ASP C 223    91.92  65.4  76.88  77.9  15.05  36.0  24.48  49.7  67.44  74.0
RES ASP C 224    47.89  34.1  47.89  48.5   0.00   0.0  13.15  26.7  34.73  38.1
RES VAL C 225     4.92   3.2   0.00   0.0   4.92  11.9   0.00   0.0   4.92  13.6
RES ALA C 226     4.14   3.8   4.14   6.5   0.00   0.0   4.14   5.8   0.00   0.0
RES GLY C 227     5.96   7.4   0.00   N/A   5.96   7.4   0.06   0.1   5.91  13.9
RES GLY C 228     0.36   0.4   0.00   N/A   0.36   0.4   0.27   0.7   0.09   0.2
RES ILE C 229    17.82  10.2  17.82  13.3   0.00   0.0  17.82  12.8   0.00   0.0
RES PHE C 230    32.43  16.3  32.43  20.1   0.00   0.0  32.43  19.6   0.00   0.0
RES HIS C 231     0.10   0.1   0.00   0.0   0.10   0.3   0.00   0.0   0.10   0.1
RES SER C 232     0.22   0.2   0.22   0.3   0.00   0.0   0.00   0.0   0.22   0.3
RES LEU C 233     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES ALA C 234     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 235     4.66   2.6   4.66   3.4   0.00   0.0   4.66   3.3   0.00   0.0
RES LYS C 236    41.51  20.7  40.12  25.3   1.39   3.3  21.64  18.7  19.87  23.6
RES ASP C 237   120.96  86.1  84.87  86.0  36.09  86.4  49.97 101.4  71.00  77.9
RES GLY C 238    17.48  21.8   0.00   N/A  17.48  21.8   9.93  26.3   7.55  17.7
RES LYS C 239    71.93  35.9  71.93  45.4   0.00   0.0  33.29  28.7  38.64  45.8
RES VAL C 240     1.99   1.3   0.00   0.0   1.99   4.8   0.00   0.0   1.99   5.5
RES ILE C 241    11.33   6.5   8.34   6.2   2.99   7.3  11.33   8.2   0.00   0.0
RES ALA C 242    17.63  16.3   0.02   0.0  17.61  40.1   0.02   0.0  17.61  48.0
RES TRP C 243    23.59   9.5  21.08  10.2   2.51   5.9  23.59  12.5   0.00   0.0
RES GLY C 244    26.80  33.4   0.00   N/A  26.80  33.4   1.21   3.2  25.59  60.0
RES ASP C 245     5.40   3.8   2.18   2.2   3.22   7.7   3.22   6.5   2.18   2.4
RES ASN C 246    64.21  44.6  48.37  47.0  15.84  38.6  21.21  46.0  43.00  44.0
RES ARG C 247   142.98  60.0 105.31  53.6  37.67  90.3  69.75  90.3  73.22  45.5
RES TYR C 248   109.40  51.5 102.93  59.1   6.47  17.0  66.02  48.6  43.38  56.8
RES LYS C 249   122.81  61.3 122.65  77.4   0.16   0.4  84.52  72.9  38.30  45.4
RES GLN C 250     0.26   0.1   0.00   0.0   0.26   0.6   0.26   0.5   0.00   0.0
RES THR C 251    40.83  29.3  32.02  32.8   8.82  21.1  32.02  42.3   8.82  13.9
RES THR C 252    64.97  46.7  54.26  55.6  10.71  25.7  56.50  74.6   8.46  13.3
RES VAL C 253    24.46  16.2   7.27   6.6  17.19  41.7   7.27   6.3  17.19  47.6
RES PRO C 254    41.61  30.6  30.18  27.7  11.43  42.2  41.61  34.5   0.00   0.0
RES THR C 255   116.85  83.9 105.84 108.5  11.01  26.4  73.12  96.5  43.73  68.9
RES GLU C 256    75.96  44.1  68.88  52.8   7.08  17.0  44.47  74.2  31.49  28.1
RES ALA C 257     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 258    73.72  41.3  63.73  45.9   9.99  25.3  64.21  45.2   9.50  26.1
RES SER C 259    62.45  53.6  62.24  85.1   0.21   0.5  35.50  73.2  26.95  39.6
RES GLY C 260    46.25  57.6   0.00   N/A  46.25  57.6  30.87  81.9  15.38  36.1
RES VAL C 261     7.18   4.7   0.00   0.0   7.18  17.4   0.07   0.1   7.11  19.7
RES SER C 262    75.37  64.7  52.71  72.0  22.65  52.2  46.99  96.8  28.38  41.7
RES ALA C 263    30.37  28.1  28.88  45.2   1.49   3.4  30.37  42.7   0.00   0.0
RES ILE C 264    13.25   7.6   0.24   0.2  13.01  31.6   0.24   0.2  13.01  36.0
RES ALA C 265    11.25  10.4   7.17  11.2   4.07   9.3  11.25  15.8   0.00   0.0
RES SER C 266    14.18  12.2   0.00   0.0  14.18  32.7   0.00   0.0  14.18  20.8
RES GLY C 267     4.26   5.3   0.00   N/A   4.26   5.3   4.26  11.3   0.00   0.0
RES GLU C 268    19.97  11.6  19.71  15.1   0.26   0.6   3.46   5.8  16.51  14.7
RES TRP C 269    71.97  28.9  71.57  34.6   0.40   0.9  55.98  29.6  15.99  26.7
RES TYR C 270     2.69   1.3   2.58   1.5   0.11   0.3   2.69   2.0   0.00   0.0
RES SER C 271     0.17   0.1   0.08   0.1   0.09   0.2   0.05   0.1   0.12   0.2
RES LEU C 272     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES ALA C 273     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 274     7.92   4.4   7.92   5.7   0.00   0.0   7.92   5.6   0.00   0.0
RES LYS C 275    34.87  17.4  33.98  21.4   0.89   2.1  14.45  12.5  20.43  24.2
RES ASN C 276   129.30  89.8  91.51  88.9  37.79  92.1  44.77  97.0  84.53  86.4
RES GLY C 277    18.75  23.3   0.00   N/A  18.75  23.3   9.93  26.3   8.82  20.7
RES LYS C 278    57.88  28.9  57.88  36.5   0.00   0.0  25.46  22.0  32.42  38.5
RES VAL C 279     2.16   1.4   0.00   0.0   2.16   5.3   0.00   0.0   2.16   6.0
RES ILE C 280    13.79   7.9  10.66   8.0   3.13   7.6  13.79   9.9   0.00   0.0
RES ALA C 281    16.27  15.1   0.00   0.0  16.27  37.0   0.00   0.0  16.27  44.3
RES TRP C 282    12.76   5.1   9.75   4.7   3.01   7.1  12.68   6.7   0.08   0.1
RES GLY C 283     5.71   7.1   0.00   N/A   5.71   7.1   0.32   0.9   5.39  12.6
RES SER C 284    41.04  35.2  18.02  24.6  23.02  53.1  21.71  44.7  19.33  28.4
RES SER C 285   115.38  99.0  76.47 104.5  38.91  89.7  55.35 114.0  60.03  88.2
RES ARG C 286    33.99  14.3  25.44  12.9   8.55  20.5   5.50   7.1  28.49  17.7
RES THR C 287   100.86  72.4  92.86  95.2   8.01  19.2  74.06  97.8  26.81  42.2
RES ALA C 288    26.61  24.7   4.84   7.6  21.78  49.6   5.20   7.3  21.41  58.3
RES PRO C 289    32.77  24.1  21.11  19.4  11.66  43.1  32.36  26.8   0.41   2.7
RES SER C 290   104.74  89.9  79.97 109.3  24.76  57.1  51.40 105.9  53.33  78.4
RES SER C 291    55.70  47.8  43.65  59.6  12.05  27.8  49.36 101.7   6.34   9.3
RES VAL C 292     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES GLN C 293    77.19  43.3  64.97  47.6  12.22  29.3  30.22  58.2  46.97  37.2
RES SER C 294    68.64  58.9  68.18  93.2   0.46   1.1  36.09  74.4  32.55  47.8
RES GLY C 295    50.23  62.5   0.00   N/A  50.23  62.5  37.88 100.5  12.35  29.0
RES VAL C 296     8.31   5.5   0.00   0.0   8.31  20.2   0.67   0.6   7.64  21.2
RES SER C 297    82.98  71.2  55.31  75.6  27.66  63.8  49.23 101.4  33.75  49.6
RES SER C 298    44.24  38.0  41.15  56.2   3.09   7.1  27.81  57.3  16.43  24.2
RES ILE C 299    19.50  11.1   2.14   1.6  17.37  42.2   2.60   1.9  16.90  46.8
RES GLU C 300    48.91  28.4  48.54  37.2   0.37   0.9   9.28  15.5  39.62  35.3
RES ALA C 301    15.08  14.0   0.73   1.1  14.35  32.7   0.73   1.0  14.35  39.1
RES GLY C 302     0.63   0.8   0.00   N/A   0.63   0.8   0.63   1.7   0.00   0.0
RES PRO C 303    19.34  14.2  14.78  13.6   4.56  16.8  17.36  14.4   1.98  13.1
RES ASN C 304    36.81  25.6  35.48  34.5   1.32   3.2   6.14  13.3  30.67  31.3
RES ALA C 305     2.06   1.9   0.79   1.2   1.28   2.9   0.79   1.1   1.28   3.5
RES ALA C 306     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES TYR C 307     0.53   0.3   0.53   0.3   0.00   0.0   0.00   0.0   0.53   0.7
RES ALA C 308     0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0   0.00   0.0
RES LEU C 309    16.79   9.4  16.79  12.1   0.00   0.0  16.79  11.8   0.00   0.0
RES LYS C 310    75.45  37.7  48.48  30.6  26.97  64.6  50.77  43.8  24.68  29.3
END  Absolute sums over single chains surface
CHAIN  1 C    10067.9       7158.2       2909.7       5505.4       4562.5
END  Absolute sums over all chains
TOTAL         10067.9       7158.2       2909.7       5505.4       4562.5
