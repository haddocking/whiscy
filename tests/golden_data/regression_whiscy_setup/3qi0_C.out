   50
MASTER         0.00000 15.13061 20.28600 20.28600 -1.00000 -1.00000 -1.00000 19.97746 -1.00000 33.39279 12.55011 26.48376 -1.00000 -1.00000 -1.00000 30.10463 78.99357 73.18011 29.43742 18.60698 35.59461 30.21520 18.56706 38.42122 35.97717 59.76880 59.76880 61.70419 51.61048 -1.00000 88.59048 -1.00000 -1.00000 -1.00000 -1.00000 22.90606 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000 35.46407 89.04275 52.25389 52.57530 51.62028 17.97505 32.46268
